doc/onboarding.md

```diff
@@ -100,18 +107,20 @@ onboarding session.
         (especially if it just has nits left).
   * Approving a change
     * Collaborators indicate that they have reviewed and approve of the
-        the changes in a pull request by commenting with `LGTM`, which stands
-        for "looks good to me".
-    * You have the power to `LGTM` another collaborator's (including TSC/CTC
-        members) work.
-    * You may not `LGTM` your own pull requests.
-    * You have the power to `LGTM` anyone else's pull requests.
+        the changes in a pull request using Github’s approval interface
+    * Some people like to comment `LGTM` (“Looks Good To Me”)
+    * You have the power to approve any other collaborator's work.
+    * You may not approve your own pull requests.
```

Comment:
I’ll pick the first part of the suggestion, the rest is kind of implicit, right? (Also, remember that there’s usually an extra audio or text stream that goes over all of this – I’m okay with not explaining everything in here, just the outlines)